sync-job/base/cron-worfklow.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: argoproj.io/v1alpha1
+kind: CronWorkflow
+metadata:
+  name: sync-job-cw
+spec:
+  schedule: "0 10 14 * *"
+  timezone: "Etc/UTC"
+  concurrencyPolicy: "Replace"
+  startingDeadlineSeconds: 0
+  workflowSpec:
+    paralellism: 4
+    entrypoint: sync-job-wt
+    templates:
+      container:
+        image: quay.io/thoth-station/sync-job:v0.2.3-dev
+        command: ["oc", "process"]
+        args: ["-p IMAGE_VERSION=$IMAGE_VERSION", "-p THOTH_SYNC_FORCE=$THOTH_SYNC_FORCE", "-p THOTH_SYNC_GRACEFUL=$THOTH_SYNC_GRACEFUL", "-p THOTH_SYNC_DEBUG=$THOTH_SYNC_DEBUG", "-f openshift.yaml", " | oc apply -f -"]
```

Comment:
This is not needed.
There is concept of source-to-image , the entrypoint is already set to the run script that can `app.py` and others based on the programming language. The pointer to different script can be done via setting env vars
https://docs.openshift.com/dedicated/3/using_images/s2i_images/python.html#using-images-python-configuration